pinc/DPage.inc

```diff
@@ -86,15 +90,15 @@ function Pages_prepForRound( $projectid, $round_number )
         // how project_add_page() sets page-states.
 
         $timestamp = time();
-
-        $result = mysqli_query(DPDatabase::get_connection(), "
+        $sql = "
```

Comment:
Is there a reason not to use `sprintf()` here?

Reply:
Not really except it was interleaved with column names. But hey, I can use setters here again!